neurons/validators/src/services/miner_service.py

```
@@ -90,21 +91,37 @@ async def request_job_to_miner(
         try:
             logger.info(_m("Requesting job to miner", extra=get_extra_info(default_extra)))
 
+            optin_info = await ValidatorPortalAPI.get_opt_in_connection_info(payload.miner_hotkey)
+            if optin_info.get("opt_in_status") is True:
+                miner_address = optin_info.get("central_miner_ip")
+                miner_port = optin_info.get("central_miner_port")
+            else:
+                miner_address = payload.miner_address
```

@LegendWorrier please raise an error if payloda.miner_address is None.

---

@LegendWorrier all miners will have the axon_info here and no need to get the optin_info again.